src/sentry/api/endpoints/team_details.py

```diff
@@ -40,12 +40,22 @@ def get(self, request, team):
         :pparam string organization_slug: the slug of the organization the
                                           team belongs to.
         :pparam string team_slug: the slug of the team to get.
+        :qparam list expand: an optional list of strings to opt in to additional
+            data. Supports `projects`, `externalTeams`.
+        :qparam list collapse: an optional list of strings to opt out of certain
+            pieces of data. Supports `organization`.
         :auth: required
         """
-        context = serialize(team, request.user)
-        context["organization"] = serialize(team.organization, request.user)
+        collapse = request.GET.getlist("collapse", [])
+        expand = request.GET.getlist("expand", [])
 
-        return Response(context)
+        # A little hack to preserve existing behavior.
+        if "organization" in collapse:
+            collapse.remove("organization")
```

Comment:
@mgaeta why have `collapse` if we're going to keep it as an empty array? I don't see it being used in the serializer.